gradoop-flink/src/main/java/org/gradoop/flink/io/impl/text/GDLConsoleOutput.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.text;
+
+import org.apache.flink.api.java.io.LocalCollectionOutputFormat;
+import org.gradoop.common.model.impl.pojo.Edge;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Allows to print a logical graph to the standard output.
+ */
+public class GDLConsoleOutput {
```

Comment:
May you explain me why it is the better approach to make the print method static?

Regarding your note: It's basically just the initialization of the instance variables. Still too much logic?

Reply:
This method don't need status nor is it useful to store the list of edges and vertices in memory for a longer time since no one would print the same stuff over and over again. The static method ensures fast garbage collection. Furthermore your constructor has logic, e.g.: "logicalGraph.getVertices().output(new LocalCollectionOutputFormat<>(vertices));" 

This is more than simple declaration.

Reply:
Okay thanks, I changed it the way you proposed.